site/examples/mentions.tsx

```diff
@@ -181,6 +182,7 @@ const Mention = ({ attributes, children, element }) => {
     <span
       {...attributes}
       contentEditable={false}
+      data-test={`mention-${element.character.replace(' ', '-')}`}
```

Comment:
`data-cy` might be a better name for this, as its only use is to be used by cypress tests specifically
```suggestion
      data-cy={`mention-${element.character.replace(' ', '-')}`}
```

Reply:
Done